go/indexer/indexer.go

```diff
@@ -128,7 +130,7 @@ func NewIndexer(cfg Config, gitVersion string) (*Indexer, error) {
 		return nil, err
 	}
 
-	l2Client, err := dialL2EthClientWithTimeout(ctx, cfg.L2EthRpc)
+	l2Client, l2RPC, err := dialL2EthClientWithTimeout(ctx, cfg.L2EthRpc)
```

Comment:
Note that if we use l2 geth for the RPC, its not going to have the http2 fix that was recently merged into upstream

Reply:
We'll need to clean this up entirely to favor the upstream Geth client.

Reply:
For now, I'd like to keep this as-is.